docs/uart.rst

```diff
@@ -0,0 +1,65 @@
+UART
+****
+
+.. py:module:: microbit.uart
+
+The ``uart`` module lets you talk to a device connected to your board using
+a serial interface.
+
+
+Functions
+=========
+
+.. py:function:: init(baudrate=9600, bits=8, parity=None, stop=1, pins=None)
+
+    Initialize serial communication with the specified parameters on the
+    specified ``pins``. Note that for correct communication, the parameters
+    have to be the same on both communicating devices.
+
+    The ``baudrate`` defines the speed of communication. Common baud
+    rates include:
+
+        * 9600
+        * 14400
+        * 19200
+        * 28800
+        * 38400
+        * 57600
+        * 115200
+
+    The ``bits`` defines the size of bytes being transmitted, it can be either
+    7 or 8. The ``parity`` parameter defines how partity is checked, and it can
+    be ``None``, ``microbit.uart.ODD`` or ``microbit.uart.EVEN``. The ``stop``
+    parameters tells the number of stop bits.
```

Comment:
Yes, either way.  If these args are not in this version then it still provides a path for upgrading to include new keyword args which is backwards compatible.  If the args are in this version then they'd need to be checked to be valid values (ie bits=8, stop=1).